packages/cli/src/create/add.ts

```diff
@@ -229,9 +229,9 @@ export async function addIndexer({
   console.log();
 
   consola.info(
-    `Before running the indexer, run ${cyan(`${options.packageManager} run install`)}${
+    `Before running the indexer, run ${cyan(`${options.packageManager} install`)}${
       language === "typescript"
-        ? " & " + cyan(`${options.packageManager} run prepare`)
+        ? cyan(` && ${options.packageManager} run prepare`)
         : ""
     }`,
   );
```

Comment:
Wrap it in single `cyan` color.

```suggestion
  consola.info(
    `Before running the indexer, run ${cyan(
      `${options.packageManager} install${
        language === "typescript"
          ? " && " + `${options.packageManager} run prepare`
          : ""
      }`,
    )}`,
  );
```

Reply:
@ncoquelet  Also can you fix logs in the end of `packages/cli/src/create/init.ts` too.

Reply:
during the merge, I see that you add a condition on package manager `${options.packageManager === "npm" ? " run" : ""}`
As all package managers support `run` command (npm, pnpm, yarn, bun), I suggest to leave  this code as it is to keep readability.

Reply:
here my last commited suggestion

```ts
const baseCommand = `${options.packageManager} install`;
const tsCommand = `${baseCommand} && ${options.packageManager} run prepare`;
consola.info(`Before running the indexer, run ${cyan(language === "typescript" ? tsCommand : baseCommand)}`);
  ```

Reply:
> during the merge, I see that you add a condition on package manager ${options.packageManager === "npm" ? " run" : ""}
As all package managers support run command (npm, pnpm, yarn, bun), I suggest to leave this code as it is to keep readability

yeah i did that change a few days back then you opened a PR so i didn't change it, so you can update your PR.

Reply:
> here my last commited suggestion

looks good!

Reply:
yes, I always rebase, isn't that what I did?